week1/chileshe.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+const http = require("http");
+
+const server = http.createServer((req, res) => {
+    console.log("Creating server");
+});
+
+server.on("connection", () => {
+    console.log("Connected")
+});
+
+
+const PORT = 8080;
```

Comment:
Good use of `const` for things that aren't going to change, and extra points for using the convention of all-caps for constants.